afew/tests/test_dkimvalidityfilter.py

```diff
@@ -0,0 +1,158 @@
+"""Test suite for DKIMValidityFilter.
+"""
+import unittest
+from email.utils import make_msgid
+from unittest import mock
+
+import dkim
+import dns.exception
+
+from afew.Database import Database
+from afew.filters.DKIMValidityFilter import DKIMValidityFilter
+
+
+class _AddTags:  # pylint: disable=too-few-public-methods
```

Comment:
I am wondering if we should centralize that sort of config in a pylintrc, but it is a lesser detail.

Reply:
well, that depends whether this is exception of the rule or actually "too-few-public-methods" warning isn't something we'd like to bothered at all. I'd say this warning is absolute non-sense.

Reply:
In some cases this makes sense, for example to detect classes with a single method which should better be a function.

IMHO, it's perfectly valid to add this for something as synthetic as test tooling.